### ISSUE TITLE
Security policy as build artefact does not allow external loading of scripts

### ISSUE THREAD

**Issue by erikgk** [gkushang/cucumber-html-reporter#101]
Can we include a reporter option to download these files & store them with the report instead of having them live links to the web? I have no control over the Jenkins build server so there is no way around this setting.

```text
Refused to load the script 'https://netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js' because it violates the following Content Security Policy directive: "default-src 'self'".
Note that 'script-src' was not explicitly set, so 'default-src' is used as a fallback.
```

**Comment by gkushang** [comment]
@erikgk 

Could you explain which files are you looking to download? Are they screenshots?

**Comment by rDr4g0n** [comment]
I have the same issue as well. I hoped the simple report would be entirely self contained, but it requires 2 other scripts just to enable the screenshot hide/show feature. Maybe we need a simple-simple report that has zero dependencies?

**Comment by gkushang** [comment]
published with v3.0.1. @rDr4g0n thanks for helping with the fix
